# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../linux26-rt/glibc-2.9.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./scripts/mod/sumversion.c.orig	2009-10-02 16:50:24.000000000 +0200
+++ ./scripts/mod/sumversion.c	2009-10-02 16:53:18.000000000 +0200
@@ -9,6 +9,7 @@
 #include <string.h>
 #include "modpost.h"
 
+#include <limits.h>
 /*
  * Stolen form Cryptographic API.
  *
